README.md

```diff
@@ -24,12 +24,34 @@ Or if you want to use the latest Feathr code from GitHub:
 pip install git+https://github.com/linkedin/feathr.git#subdirectory=feathr_project
 ```
 
-## Quick Start
+## Running Feathr with 3 Simple Steps
+
+Feathr has native cloud integration and getting started with Feathr is very straightforward. You only need three steps:
+
+1. Get the principal ID of your account by running `az ad signed-in-user show --query objectId -o tsv` in the link below (Select "Bash" if you are asked to choose one), and write down that value (will be something like `b65ef2e0-42b8-44a7-9b55-abbccddeefff`)
+
+[![Launch Cloud Shell](https://shell.azure.com/images/launchcloudshell.png "Launch Cloud Shell")](https://shell.azure.com/bash)
```

Comment:
It's because of this issue:
https://github.com/MicrosoftDocs/azure-docs/issues/91284

but anyway I just removed the image to minimize the impact. Will add the image back when Azure team fixed this.